modules/cluster-logging-rn-5.6.2.adoc

```diff
@@ -0,0 +1,29 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-6-2_{context}"]
+= Logging 5.6.2
+This release includes link:https://access.redhat.com/errata/RHBA-2023:0793[OpenShift Logging Bug Fix Release 5.6.2].
+
+[id="openshift-logging-5-6-2-bug-fixes"]
+== Bug fixes
+* Before this update, the collector did not set `level` fields correctly based on priority for systemd logs. With this update, `level` fields are set correctly. (link:https://issues.redhat.com/browse/LOG-3429[LOG-3429])
+
+* Before this update, the Operator incorrectly generated incompatibility warnings on  {product-title} 4.12+. With this update, the Operator max {product-title} version value has been corrected, resolving the issue. (link:https://issues.redhat.com/browse/LOG-3584[LOG-3584])
+
+* Before this update, creating a `ClusterLogForwarder` custom resource (CR) with an output value of `default` did not generate any errors. With this update, an error warning this value is invalid generates appropriately. (link:https://issues.redhat.com/browse/LOG-3437[LOG-3437])
+
+* Before this update, when the `ClusterLogForwarder` custom resource (CR) had multiple pipelines configured, with one output set as `default`, the collector pods restarted. With this update, the logic for output validation has been corrected, resolving the issue. (link:https://issues.redhat.com/browse/LOG-3559[LOG-3559])
+
+* Before this update, collector pods restarted after being created. With this update, the deployed collector does not restart on its own. (link:https://issues.redhat.com/browse/LOG-3608[LOG-3608])
+
+* Before this update, patch releases removed previous versions of the operators from the catalog. This made installing the old versions impossible. This update changes bundle configurations so that previous releases of the same minor version stay in the catalog. (link:https://issues.redhat.com/browse/LOG-2730[LOG-2730])
```

Comment:
Sorry, there has been another last-minute change. LOG-2730 was replaced by LOG-3635. Message stays the same.

```suggestion
* Before this update, patch releases removed previous versions of the operators from the catalog. This made installing the old versions impossible. This update changes bundle configurations so that previous releases of the same minor version stay in the catalog. (link:https://issues.redhat.com/browse/LOG-3635[LOG-3635])
```